u19_pipeline/acquisition.py

```diff
@@ -29,7 +29,8 @@ class Session(dj.Manual):
     session_start_time   : datetime                     # start time
     session_end_time=null : datetime                     # end time
     -> lab.Location.proj(session_location="location")
-    -> task.TaskLevelParameterSet
+    -> task.TaskLevelParameterSet.proj(last_level="level")
+    -> [nullable] lab.ManipulationType.proj(manipulation_type="manipulation_type") # The type of manipulation
```

Comment:
this is not necessary for this PR, but is a suggestion for the future. We can ignore it for this PR.